package/libs/openssl/Makefile

```diff
@@ -352,6 +354,14 @@ ifdef CONFIG_i386
   endif
 endif
 
+ifndef CONFIG_OPENSSL_WITH_QUIC
+  OPENSSL_OPTIONS += no-quic
+endif
+
+ifndef CONFIG_OPENSSL_WITH_KTLS
+  OPENSSL_OPTIONS += no-ktls
+endif
```

Comment:
This just deactivates the option when `CONFIG_OPENSSL_WITH_KTLS` is not set. Is it activates by default?
Do you need an else case like?
```
ifndef CONFIG_OPENSSL_WITH_KTLS
  OPENSSL_OPTIONS += no-ktls
else
  OPENSSL_OPTIONS += ktls
endif
```